README.md

```diff
@@ -121,13 +121,40 @@ If you use [project-stub](https://bem.info/tutorials/project-stub/) to start you
   * Safari *(the last two stable versions)*
   * Yandex *(the last two stable versions)*
   * Opera 12.6+
-  * Internet Explorer 10+
+  * Internet Explorer 9+
+  * Partially supported of Internet Explorer 8 ([more info](#ie8))
 
 * **Touch**
   * Android 4+
   * iOS 5+
   * Internet Explorer 10+
 
+<a name="ie8"></a>
+## Support IE8
+
+To support this browser you need to connect `es5-shim` and additional styles with the extension `*.ie.styl`. To do this, add to the block `page` conditional comments as shown below:
```

Comment:
To support IE8, you must connect `es5-shim` and additional styles with the `*.ie.styl` extension to the page. To do this add the conditional comments to the `page` block, as shown below: